### PR TITLE
TRIDENT-18006 - MITRE coverage

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a new article:
https://help.sumologic.com/docs/cse/administration/mitre-coverage/

Issue number:  [TRIDENT-18006](https://sumologic.atlassian.net/browse/TRIDENT-18006)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
This PR is in draft until all SME comments are done.

File: docs/cse/administration/mitre-coverage.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+         * Medium (14-16 rules)
+         * Low (11-13 rules)
+         * None (10 or fewer rules)
+         * Not detectable (no rules found)
+
+## MITRE details
+
+When you click a square in the tactics/techniques matrix, details about coverage for that tactic/technique display in a panel. The panel has a full description of the tactic/technique, with an assessment of your coverage (None, Low, Medium, and High). A coverage of "None" does not mean you have no coverage, it only means you may not have enough rules to adequately cover the tactic/technique.
+
+Click **Rules** at the bottom of the panel to get links to all the rules that contribute to coverage for the tactic/technique.
```
Comment (kimsauce) [inline docs/cse/administration/mitre-coverage.md:46]:
```suggestion
When you click a square in the tactics/techniques matrix, details about coverage for that tactic/technique display in a panel. The panel has a full description of the tactic/technique, with an assessment of your coverage (**None**, **Low**, **Medium**, and **High**). A coverage of **None** does not mean you have no coverage; it only means you might not have enough rules to adequately cover the tactic/technique.

Click **Rules** at the bottom of the panel to get links to all the rules that contribute to coverage for the tactic/technique.  
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
one minor thing

**Comment by jpipkin1** [comment]
Good suggestion, @kimsauce.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@pkazmir-sumo, could you review this PR?

**Pull request merged**